Source/items.cpp

```diff
@@ -2468,10 +2465,10 @@ void GetItemStr(int i)
 	int nGold;
 
 	if (item[i]._itype != ITYPE_GOLD) {
-		if (!item[i]._iIdentified)
-			strcpy(infostr, item[i]._iName);
-		else
+		if (item[i]._iIdentified)

```

Comment:
Interesting, I had thought that there might be a case that if's are flipped depending on alignments.

Reply:
I don't think that `if` could be flipped since order of the cases provides a suggestion for optimizer. However among modern compilers I know the case when clang flips `if (!b)` in case if `b` is `bool`.